modules/drivers/mongo/test/metabase/driver/mongo/query_processor_test.clj

```diff
@@ -111,6 +118,33 @@
                                 s/Keyword s/Any}
                                (qp/process-query (mt/native-query query)))))))))))))
 
+(deftest gruping-with-timezone-test
+  (mt/test-driver :mongo
+    (testing "Result timezone is respected when grouping by hour (#11149)"
+      (mt/dataset attempted-murders
+        (testing "Querying in UTC works"
+          (mt/with-system-timezone-id "UTC"
+            (is (= [["2019-11-20T20:00:00Z" 1]
+                    ["2019-11-19T00:00:00Z" 1]
+                    ["2019-11-18T20:00:00Z" 1]
+                    ["2019-11-17T14:00:00Z" 1]]
+                   (mt/rows (mt/run-mbql-query attempts
+                              {:aggregation [[:count]]
+                               :breakout [[:field %datetime {:temporal-unit :hour}]]
+                               :order-by [[:desc [:field %datetime {:temporal-unit :hour}]]]
+                               :limit 4}))))))
+        (testing "Querying in Kathmandu works"
+          (mt/with-system-timezone-id "Asia/Kathmandu"
+            (is (= [["2019-11-21T01:00:00+05:45" 1]
```

Comment:
I see we aren't doing the _true_ hipster Kathmandu tz, which is +05:41:16 (let's not do the _true_ hipster Kathmandu tz, lol)

Reply:
Dumb question but if Kathmandu was _really_ +05:41:16 in 2019 is there a chance this test might break in the future if they update the JRE timezone info? Might be worth adding a little comment or `testing` context here so if this test breaks in the future people won't have to stare at it in confusion for a long time